juju/juju-qa-jenkins
# [JUJU-1320] add vsphere provider to integration tests

Using the VSphere in Boston

Also enable integration tests for Vsphere for the network tests

This PR depends on the (as of yet unmeregd) PRs:
https://github.com/juju/juju-qa-jenkins/pull/28
https://github.com/juju/juju-qa-jenkins/pull/31

## QA Steps

verify that the network test passes in VSphere locally

```
BOOTSTRAP_PROVIDER=vsphere BOOTSTRAP_CLOUD=vsphere-boston BOOTSTRAP_REGION=Boston ./main.sh network
```

Also verify that the job passes in Jenkins:
https://jenkins.juju.canonical.com/job/test-network-vsphere-boston/
